File: Demo-LLP-MIP/Demo-LLP-MIP-Python/v1/isign/argument/apply/ApplyParams.py
from v1.info.CardInfo import CardInfo


class ApplyParams:
    def __init__(self):
        self.mch_id = None
        self.sub_mchid = None
        self.user_id = None
        self.txn_seqno = None
        self.txn_time = None
        self.card_info = None  # 假设 CardInfo 为另一个类

    def to_dict(self):
        return {
            "mch_id": self.mch_id,
            "sub_mchid": self.sub_mchid,
            "user_id": self.user_id,
            "txn_seqno": self.txn_seqno,
            "txn_time": self.txn_time,
            "card_info": self.card_info.__dict__ if self.card_info is not None else None
        }

File: Demo-LLP-MIP/Demo-LLP-MIP-PHP/v1/isign/argument/apply/ApplyParams.php
<?php


class ApplyParams
{
    public $mch_id;
    public $sub_mchid;
    public $user_id;
    public $txn_seqno;
    public $txn_time;
    public $card_info;

    public function __construct()
    {
        $this->mch_id = null;
        $this->sub_mchid = null;
        $this->user_id = null;
        $this->txn_seqno = null;
        $this->txn_time = null;
        $this->card_info = null;  // 假设 CardInfo 为另一个类
    }

    public function to_dict() {
        $data = [];
        foreach ($this as $key => $value) {
            if ($value !== null) {
                $data[$key] = $value;
            }
        }
        return json_encode($data);
    }
}